### PR TITLE
Emily carter static review

### PR DESCRIPTION
**Pull request by kauket**
Static Review Request

### CODE REVIEW

File: src/Book.java [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
     public String toString() {
-        StringBuilder stringBuilder = new StringBuilder();
-        stringBuilder.append("book: ").append(bookId).append("\n")
-                .append("  Title:  ").append(title).append("\n")
-                .append("  Author: ").append(author).append("\n")
-                .append("  CallNo: ").append(callNo).append("\n")
-                .append("  State:  ").append(state);
-        return stringBuilder.toString();
+        return "book: " + bookId + "\n" +
+                "  Title:  " + title + "\n" +
```
Comment (johngalvin09) [inline src/Book.java]: Line 26 to 29 are not indented with an increment of 4 spaces. They are currently double-indented

Comment (kauket) [reply src/Book.java]: "When there are multiple continuation lines, indentation may be varied beyond +4 as desired." Google Java Style Guide 4.5.2

Comment (johngalvin09) [reply src/Book.java]: Confirmed and accepted.

**Comment by MethodicalMadness** [comment]
Mediation: Confirmed code was working after merge.
Michael Coleman

**Pull request merged**